defaults/main.yml

```diff
@@ -18,7 +18,7 @@ __server_port: 8085
 
 __java_version: 17.0.3_7
 
-__nginx_version: 1.19.8
+__nginx_version: 1.24.0
```

Comment:
Is this required?

Reply:
yes, this version is tested and available on both CentOS and Ubuntu